programs/bpf_loader/src/serialization.rs

```diff
@@ -40,7 +40,7 @@ struct Serializer {
 impl Serializer {
     fn new(size: usize, start_addr: u64, aligned: bool, copy_account_data: bool) -> Serializer {
         Serializer {
-            buffer: AlignedMemory::with_capacity(size),
+            buffer: AlignedMemory::with_capacity_zeroed(size),
```

Comment:
Why would zeroing improve perf? In anything it should be slower

Reply:
It is faster because fill_write during account serialization is already initialized so only set_len for vector is called, where as if we only increase the capacity vector resize is called which inturn call vector extend with value 0 which seems to be more expensive.